splash/browser_tab.py

```diff
@@ -892,6 +904,12 @@ def _send_request(self, url, callback, method='GET', body=None,
 
         request = self.request_obj(url, headers=headers, body=body)
 
+        if headers and "user-agent" in [k.lower() for k in headers.keys()]:
+            user_agent = headers.get("user-agent", headers.get("User-Agent"))
+        else:
+            user_agent = self.web_page.userAgentForUrl(to_qurl(url))
+        request.setRawHeader(b"user-agent", to_bytes(user_agent))
```

Comment:
Content-Type header is set in self.request_obj; is there a reason User-Agent header is not set there as well?